My Survey-results2022-10-21.csv

```diff
@@ -0,0 +1 @@
+username,course,Q7: Long Text Question
```

Comment:
This is a file generated during a test run that didn't get deleted. (This can happen sometimes when a test is stopped before the cleanup routine that should delete this file gets run.)